src/Verb.js

```diff
@@ -120,7 +120,7 @@ TinCan client library
 
                 if (this.display === null && typeof _downConvertMap[this.id] !== "undefined") {
                     this.display = {
-                        "und": _downConvertMap[this.id]
+                        "en": _downConvertMap[this.id]
```

Comment:
True, but at the same time we've discussed adding the ability to set the language accept header and it could be argued that there is very little reason outside of a debugging context to have a browser request a statement stream (and even in a debugging context).

Whether it goes away with dropping 0.9 is based on how open we are to backwards compatibility breakage, and with the roadmap to 1.x we are very open to it as a major version bump. The conversion of ids is something I definitely want to see go, if developers are using the short cuts just to get the ADL verbs it should be easy for them to introduce the conversion map in their own code to handle it, never the other OSS libs currently do any of these types of things and developers don't seem to have a problem using a proper verb ID.